apps/desktop/src/panes/PaneManager.tsx

```
@@ -7,42 +7,61 @@ import { HistoryPane } from "./HistoryPane";
+
 export const PaneManager: React.FC = () => {
   const { panes, activePaneId } = usePaneStore();
 
   // Get data from global object (temporary solution)
-  const data = (window as any).__openagents_data || {};
+  const data = window.__openagents_data;
 
   const renderPaneContent = (pane: PaneType) => {
     switch (pane.type) {
       case "chat":
         const sessionId = pane.content?.sessionId as string;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)